i18n/en/docusaurus-plugin-content-docs/current/intro.mdx

```diff
@@ -98,7 +19,7 @@ import Link from "@docusaurus/Link";
     description="A tour over the basic concepts and structure as well as a comprehensive review of a React sample project"
     to="/docs/get-started"
     Icon={"🚀"}
-    tags={['Basics', 'Quick start', 'FAQ']}
+    tags={['Overview', 'Quick start', 'FAQ']}
```

Comment:
> Style the cards in the index page differently. Make them fun and colorful, like in Gatsby, arrange them in a grid and cut down the amount. 2 cards is the ideal amount, honestly :)

Again, we have #445 issue for this

Redesigning navcards and docs sections - it more difficult task 🤔 
(unless you have "easy to fix/impl" suggestions)

Reply:
Also, we couldn't decrease amount right now because:
- We have incomplete restructuring of doc (#263)
   (we will discuss it detailed at next DOCS sync)
- Without dropdown, this page will be the only one entry-point for docs navigating
   (it is why its `IndexPage`)

Reply:
> Without dropdown, this page will be the only one entry-point for docs navigating

Notice that the Gatsby index page is also quite packed with links and items, but it works, because only two of them are heavily prominent, the rest is just quiet links below. I suggest we do the same, clearly delineate the priorities

Reply:
> Redesigning navcards and docs sections - it more difficult task 🤔

Agreed, it's not as simple. However, I think that if we don't do it, but still apply this PR, we will be in a risky transitive state where the navigation may actually be worse for a while. So maybe we could do them together as a unit of work, with a bit more planning and time devoted to it?

Reply:
> I suggest we do the same, clearly delineate the priorities

If u suggest these navcards - I don't mind

<img width="1084" alt="image" src="https://user-images.githubusercontent.com/42924400/189478212-41e969d5-73d0-4a0d-b6ca-4c0ab7f08d5d.png">

But I think user also could be "confused" as u say, if we suggest him more choices and we'll again get effect "make me afraid as a reader of missing out." 😏

Reply:
I should point out that this effect of fear of missing out only happens when the options appear equally important, for example, when they are visually the same. When there are two prominent options and the rest is very quiet, I'm only concerned about the two prominent ones and the rest I label in mind as "reference material when I'll have exact questions"

Reply:
fixed